### DISCUSSION TITLE
Can't ping host, google.com, etc. on client

### DISCUSSION THREAD

**Discussion by glsmith86** [strongswan/strongswan#2104] [General]
My client can't ping host, internet host, speedtest.net can't find server, facebook app can't load contents. Here is my all confs and logs, what can I make in the system.

[charon.conf.txt](https://github.com/strongswan/strongswan/files/14319561/charon.conf.txt)
[strongswan.conf.txt](https://github.com/strongswan/strongswan/files/14319562/strongswan.conf.txt)
[swanctl.conf.txt](https://github.com/strongswan/strongswan/files/14319563/swanctl.conf.txt)

[ifconfig.txt](https://github.com/strongswan/strongswan/files/14319564/ifconfig.txt)
[ipsec_statusall.txt](https://github.com/strongswan/strongswan/files/14319566/ipsec_statusall.txt)
[iptables-l.txt](https://github.com/strongswan/strongswan/files/14319567/iptables-l.txt)
[iptables-save.txt](https://github.com/strongswan/strongswan/files/14319568/iptables-save.txt)

This can be show when I made command: ip addr add 10.10.10.0/32 dev ipsec0.

[iprouteshow.txt](https://github.com/strongswan/strongswan/files/14319579/iprouteshow.txt)

Sysctl ipv4 settings:
[sysctl.txt](https://github.com/strongswan/strongswan/files/14319592/sysctl.txt)

**Comment by tobiasbrunner** [comment]
Including `swanctl.conf` from `strongswan.conf` is definitely not correct. Neither is using [`load` statements](https://docs.strongswan.org/docs/5.9/plugins/pluginLoad.html#_static_load_list) and the settings in `charon.conf` obviously have no effect if that file is not included in `strongswan.conf`.

Then, why are you using the `kernel-libipsec` plugin?

Anyway, you have configured `local_ts = 192.168.1.0/24` so only traffic to that subnet will be accepted from the client. That obviously doesn't include any hosts on the Internet. If you want all traffic from the client sent to your server configure `local_ts = 0.0.0.0/0`. If you actually want split-tunneling but it doesn't work, then that might be a limitation of the client.

Regarding traffic from the client, [follow the docs on forwarding traffic](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html).

**Reply by glsmith86** [reply]
Include swanctl.conf in strongswan.conf because ipsec start command not load swanctl.conf file. It is solved. 
If I use 0.0.0.0/0 as traffic selector, my ad-blocker on  the router not filter the traffic. If I use 192.168.1.0/24, I can't connect to Internet.

kernel-libipsec plugin deleted from the loading list.

**Reply by tobiasbrunner** [reply]
> Include swanctl.conf in strongswan.conf because ipsec start command not load swanctl.conf file.

Well, that's not how to do that.  But I guess you figured that out in the mean time.

> If I use 0.0.0.0/0 as traffic selector, my ad-blocker on the router not filter the traffic.

What do you mean? What ad-blocker, what router?

> If I use 192.168.1.0/24, I can't connect to Internet.

Well, obviously, because no traffic to the Internet is actually tunneled via VPN.

**Reply by glsmith86** [reply]
> Well, that's not how to do that. But I guess you figured that out in the mean time.

First time I'm place start-scripts in charon.conf file. I moved it to strongswan.conf file.

> What do you mean? What ad-blocker, what router?

I have an Asus RT-AX56U router with AsusWRT Merlin system, it run Diversion ad-blocker.  It works well when I'm at home and my phone connect to wifi.

**Reply by tobiasbrunner** [reply]
> I have an Asus RT-AX56U router with AsusWRT Merlin system, it run Diversion ad-blocker. It works well when I'm at home and my phone connect to wifi.

So what does that have to do with the VPN?

**Comment by glsmith86** [comment]
> So what does that have to do with the VPN?

I want to use to reach my home network with sharings an block ads when I'm not at home.

**Reply by tobiasbrunner** [reply]
Then just follow the docs on forwarding traffic I linked before (and you probably want to configure your router as DNS server if the ad blocker is DNS-based, but keep in mind that there could be conflicts with private networks).
